bindata/assets/deployments/console-deployment.yaml

```diff
@@ -70,11 +70,15 @@ spec:
               path: /health
               port: 8443
               scheme: HTTPS
-            initialDelaySeconds: 150
-            timeoutSeconds: 1
+            failureThreshold: 1
             periodSeconds: 10
-            successThreshold: 1
-            failureThreshold: 3
+          startupProbe:
```

Comment:
I recommend explicitly setting all the various values

```
      initialDelaySeconds: 0
      periodSeconds: 5
      timeoutSeconds: 10
      successThreshold: 1
      failureThreshold: 30
```

I'm less critical of the content, but future owners will likely thank you for explaining their various options even if they are the defaults.